canvas/services/event.py

```diff
@@ -73,6 +74,14 @@ def set_featured(event):
 
 
 def add_question_set(event, category_id, difficulty, number_of_questions):
+    def begins_with_number(string):
+        # return any(char.isdigit() for char in string)
+        return string[0].isdigit()
```

Comment:
Let's not do this. If you are searching for numbers in the title it doesn't have to be at the beginning.

Reply:
you can remove this function.